docs/user-guide/commands/argocd_app_logs.md

```diff
@@ -9,9 +9,9 @@ argocd app logs APPNAME [flags]
 ### Options
```

Comment:
This one is generated by  go run tools/cmd-docs/main.go. I need to run this because it need to update the doc with the new flag output added for app resources and new option tree in output flag for app get

Reply:
I know it is generated but as it was modified it shows that you changed something unexpected